lib/modules/platform/github/index.ts

```diff
@@ -1402,7 +1422,7 @@ export async function createPr({
   );
   const { number, node_id } = ghPr;
   await addLabels(number, labels);
-  await tryPrAutomerge(number, node_id, platformOptions);
+  await tryPrAutomerge(number, node_id, platformOptions, platformConfig);
```

Comment:
why do we need to pass a global variable around? just for testing is bad 😕

Reply:
I previously tried to archive this without exporting or passing the global variable around, but I couldn't get it working and my attempts got so complex, that I though that this is solution is easier to maintain and understand in the long run.

Reply:
I got it working in [bdd4757](https://github.com/renovatebot/renovate/pull/15924/commits/bdd47576547370870e15602240b850ee9f65b3a8) although I'm not sure how "clean" that is. I'm highly interested in your opinion/feedback on that and improvement ideas.